src/main/java/uk/ac/cam/cl/dtg/segue/scheduler/SegueJobService.java

```diff
@@ -150,7 +150,6 @@ public void registerScheduleJob(final SegueScheduledJob jobToRegister) throws Sc
       log.info(String.format("Registered job (%s) to segue job execution service. Current jobs registered (%s): ",
           jobToRegister.getJobKey(), localRegisteredJobs.size()));
     } else {
-      // FIXME - this does not update the job details, e.g. if the SQL file name changes.
```

Comment:
That comment has been present since early 2019. Since there don't seem to have been any issues from it, we assumed that the 'missing' functionality is not currently needed. I checked with Dapo and it doesn't sound like we're expecting to need it in the foreseeable future either?
As such, our decision was to remove to TODO and we can revisit the code in future if it becomes relevant.
I suppose it might make sense to remove the TODO but leave the actual comment in place as a warning? I'll wait for your opinion on that.